cfgov/preprocessed/css/organisms/post-preview.less

```diff
@@ -0,0 +1,119 @@
+            </div>
+            <div class="o-post-preview_content">
+                <h3 class="o-post-preview_title">
+                    This is a blog post
+                </h3>
+                <p class="o-post-preview_description">
+                    Sed senectus feugiat arcu tempor, ac vulputate sem pulvinar porta hendrerit magnis, interdum habitasse taciti.
+                </p>
+                <div class="post_meta">
+                    <span class="o-post-preview_byline">
+                        By <a href="/blog/?filter_author=Tom">
+                            Tom
+                        </a> and
+                        <a href="/blog/?filter_author=Tom">
+                            Jerry
+                        </a>
+                    </span>
+                </div>
+                <div class="tags tags__hide-heading">
+                    <span class="u-visually-hidden">Topics:</span>
+                    <ul class="tags_list">
+                        <li class="tags_tag">
+                            <a class="tags_link" href="/blog/?filter_tags=FOIA&amp;filter_topics=FOIA">
+                                <span class="tags_bullet" aria-hidden="true">•</span>
+                                FOIA
+                            </a>
+                        </li>
+                        <li class="tags_tag">
+                            <a class="tags_link" href="/blog/?filter_tags=Mortgage&amp;filter_topics=Mortgage">
+                                <span class="tags_bullet" aria-hidden="true">•</span>
+                                Mortgage
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+          </div>
+      </article>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          article.o-post-preview
+            div.meta-header
+            div.o-post-preview_image-container
+                img.o-post-preview_image
+            div.o-post-preview_content
+                h3.o-post-preview_title
+                p.o-post-preview_description
+            p.post_dek
+            div.post_meta
+              span.post_byline
+              span.post_date
+              div.post_meta
+                span.o-post-preview_byline
+                div.tags
+  tags:
+    - cfgov-organisms
+*/
+
+.o-post-preview {
+  &_image-container {
+    .respond-to-max(@bp-sm-max, {
+      margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
+    });
+
+    .respond-to-min(@bp-med-min, {
+      .grid_column(4);
+    });
+  }
+  &_image {
+    display: block;
+    width: 100%;
+
+
+  }
+}
+
+.o-post-preview_image-container + .o-post-preview_content {
+    .respond-to-min(@bp-med-min, {
+      .grid_column(8);
+    });
+}
+
+.o-post-preview_subtitle {
+  .h6();
+}
+
+.o-post-preview_description,
+.o-post-preview_byline {
+  .webfont-regular();
```

Comment:
I've been setting the regular font family on the molecule/organism to avoid having to set it in multiple places. Should we limit it or is it ok to set scoped to a molecule/organism?